topics/rediscli.md

```diff
@@ -345,6 +345,22 @@ key, like in the following example:
     127.0.0.1:6379> ZADD<TAB>
     127.0.0.1:6379> ZCARD<TAB>
 
+Once you've entered a valid command name at the prompt, its syntax will be
+displayed by the CLI by default. The default syntax hints display can be
+controlled via the CLI's preferences.
+
+## Preferences
+
+The CLI's behavior can be customized to your preferences. Once started, the CLI
```

Comment:
Perhaps:

"There are two ways to customize the CLI's behavior.  The file `.redisclirc` in your home directory is loaded by the CLI on startup.  Preferences can also be set during a CLI session, in which case they will last only the the duration of the session."

Reply:
Accepted